[2025-04-09T17:00:38.449+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-09T17:00:38.462+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.is_weather_api_ready manual__2025-04-09T17:00:33.341764+00:00 [queued]>
[2025-04-09T17:00:38.473+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.is_weather_api_ready manual__2025-04-09T17:00:33.341764+00:00 [queued]>
[2025-04-09T17:00:38.474+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 3
[2025-04-09T17:00:38.499+0000] {taskinstance.py:2890} INFO - Executing <Task(HttpSensor): is_weather_api_ready> on 2025-04-09 17:00:33.341764+00:00
[2025-04-09T17:00:38.510+0000] {standard_task_runner.py:72} INFO - Started process 34986 to run task
[2025-04-09T17:00:38.516+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'weather_dag', 'is_weather_api_ready', 'manual__2025-04-09T17:00:33.341764+00:00', '--job-id', '106', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmppobtiju0']
[2025-04-09T17:00:38.518+0000] {standard_task_runner.py:105} INFO - Job 106: Subtask is_weather_api_ready
[2025-04-09T17:00:38.578+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_dag.is_weather_api_ready manual__2025-04-09T17:00:33.341764+00:00 [running]> on host ip-172-31-10-226.eu-north-1.compute.internal
[2025-04-09T17:00:38.702+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='myemail@domain.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='is_weather_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T17:00:33.341764+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T17:00:33.341764+00:00'
[2025-04-09T17:00:38.704+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-09T17:00:38.704+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-09T17:00:38.705+0000] {logging_mixin.py:190} INFO - Current task name:is_weather_api_ready state:running start_date:2025-04-09 17:00:38.462842+00:00
[2025-04-09T17:00:38.705+0000] {logging_mixin.py:190} INFO - Dag name:weather_dag and current dag run status:running
[2025-04-09T17:00:38.705+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-09T17:00:38.705+0000] {baseoperator.py:423} WARNING - HttpSensor.execute cannot be called outside TaskInstance!
[2025-04-09T17:00:38.706+0000] {http.py:145} INFO - Poking: /data/2.5/weather?q=egypt&APPID=4d13f07efe491003cd53c704f189b00f
[2025-04-09T17:00:38.714+0000] {base.py:84} INFO - Retrieving connection 'weathermap_api'
[2025-04-09T17:00:38.821+0000] {base.py:339} INFO - Success criteria met. Exiting.
[2025-04-09T17:00:38.828+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-09T17:00:38.829+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=is_weather_api_ready, run_id=manual__2025-04-09T17:00:33.341764+00:00, execution_date=20250409T170033, start_date=20250409T170038, end_date=20250409T170038
[2025-04-09T17:00:38.866+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-09T17:00:38.866+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-09T17:00:38.867+0000] {logging_mixin.py:190} INFO - Dag name:weather_dag queued_at:2025-04-09 17:00:33.354327+00:00
[2025-04-09T17:00:38.867+0000] {logging_mixin.py:190} INFO - Task hostname:ip-172-31-10-226.eu-north-1.compute.internal operator:HttpSensor
[2025-04-09T17:00:38.893+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-09T17:00:38.929+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-09T17:00:38.936+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
